ida/data.yml

```diff
@@ -1147,12 +1168,130 @@ classes:
       0x140B1D390: HasIndividualWeather
       0x140B1D440: GetWeatherForHour
       0x140B1D620: GetWeatherForDaytime
+  Client::Game::FashionCheckManager: # Fashion Report
+    instances:
+      - ea: 0x1421FF160
+        pointer: True
+    funcs:
+      0x1408A8700: HasInstance
+      0x1408A8720: GetInstance
+      0x1408A85F0: Create
+      0x1408A86D0: Destroy
+  Client::Game::CustomTodoManager: # Studium Deliveries
+    vtbls:
+      - ea: 0x141B64740
+        base: Client::Game::ServerRequestCallbackInterface
+    instances:
+      - ea: 0x142238B50
+        pointer: True
+    funcs:
+      0x141430480: HasInstance
+      0x141430470: GetInstance
+      0x1414304A0: Create
+      0x141430500: Destroy
```

Comment:
Should be named `dtor`

Reply:
I reversed this because the dtor is part of the vtable (@ 0x141B64740).